daemon/cmd/daemon_main.go

```diff
@@ -1642,6 +1649,33 @@ func initKubeProxyReplacementOptions() {
 			option.Config.NodePortMode = option.NodePortModeSNAT
 		}
 	}
+
+	if option.Config.EnableSessionAffinity {
+		if !probesManager.GetMapTypes().HaveLruHashMapType {
+			msg := "SessionAffinity feature requires BPF LRU maps"
```

Comment:
Is it even possible to run it on a kernel without LRU map?

Reply:
On 4.9 I saw an error (iirc invalid type) when creating a LRU map.